Add detail to searchable snapshot limits section (#6828)
opensearch-project/documentation-website: _tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot.md

- Accessing data from a remote repository is slower than local disk reads, so higher latencies on search queries are expected.
- Many remote object stores charge on a per-request basis for retrieval, so users should closely monitor any costs incurred.
- Searching remote data can impact the performance of other queries running on the same node. We recommend that users provision dedicated nodes with the `search` role for performance-critical applications.

> - Consider [force merging]({{site.url}}{{site.baseurl}}/api-reference/index-apis/force-merge/) indexes to a smaller number of segments before snapshotting for best search performance. For best performance, at the cost of using compute resources prior to snapshotting, force merge your index to one segment.

Reviewer: ```suggestion
- For better search performance, consider [force merging]({{site.url}}{{site.baseurl}}/api-reference/index-apis/force-merge/) indexes to a smaller number of segments before taking a snapshot. For best performance, at the cost of using compute resources prior to snapshotting, force merge your index to one segment.
```

> - It is recommended to configure a maximum remote data to local disk cache ratio with the `cluster.filecache.remote_data_ratio` setting. A ratio of 5 is a good starting point for most workloads to ensure good query performance. If the ratio is too large, then there may not be sufficient disk space to handle the search workload. See issue [#11676](https://github.com/opensearch-project/OpenSearch/issues/11676) for a known bug related to this scenario.

Reviewer: ```suggestion
- We recommend to configure a maximum remote data to local disk cache ratio using the `cluster.filecache.remote_data_ratio` setting. A ratio of 5 is a good starting point for most workloads to ensure good query performance. If the ratio is too large, then there may not be sufficient disk space to handle the search workload. See issue [#11676](https://github.com/opensearch-project/OpenSearch/issues/11676) for a known bug related to this scenario.
```

Reviewer 2: I think that technically "remote-data-to-local-disk-cache" should be hyphenated because it all modifies "ratio," but that's obviously not ideal, so is there a way to rephrase to avoid this?

Reviewer 2: "We recommend configuring a maximum ratio of remote data to local disk cache [size/capacity]"?

Author: "We recommend configuring a maximum ratio of remote data to local disk cache size" works for me
